jsx/Form.js

```diff
@@ -976,6 +976,27 @@ class DateElement extends Component {
     this.handleChange = this.handleChange.bind(this);
   }
 
+  componentDidMount() {
+    if (!Modernizr.inputtypes.month) {
+      let monthInputs = $('input[type=month]');
+      monthInputs.datepicker({
+        dateFormat: 'yy-mm',
+        changeMonth: true,
+        changeYear: true,
+        yearRange: '1900:' + new Date().getFullYear(),
```

Comment:
unless I'm misunderstanding, isn't this going to limit the year to the current year? I don't think this happens with things not using the polyfill. What if you need to enter a date in the future?